pillar/base/haproxy.sls

```diff
@@ -67,6 +67,22 @@ haproxy:
       verify_host: planet.psf.io
       check: "HEAD / HTTP/1.1\\r\\nHost:\\ planet.psf.io"
 
+    {% for tracker, config in pillar["bugs"]["trackers"].items() %}
```

Comment:
This needs fixin'

Reply:
maybe just a 
```
include:
  - bugs
```

Reply:
that's an option. we can also just write them out as we have for others like codespeed. matter of preference.